src/components/MarkDown/MarkdownEditor/MarkDownEditor.tsx

```diff
@@ -0,0 +1,92 @@
+import type { FC } from 'react';
+import type { MarkDownEditorProps } from './MarkDownEditor.types';
+
+import React from 'react';
+import classNames from 'classnames/bind';
+import MDEditor, { ICommand } from '@uiw/react-md-editor';
+import {
+  MARKDOWN_EDITOR_TYPES,
+	MARKDOWN_EDITOR_MODES,
+	MARKDOWN_EDITOR_TOOLBAR_TITLES
+} from './MarkDownEditor.constants';
+import styles from './MarkDownEditor.module.scss';
+
+const cx = classNames.bind(styles);
+
+export const MarkDownEditor: FC<MarkDownEditorProps> = ({
+  type,
+	text,
+	placeholder,
+	onChange,
+  error,
+	errorText,
+	className
+}) => {
+	const onCommandsFilter = (command: ICommand<string>) => {
```

Comment:
Could maybe extract `onCommandsFilter` to a `utils` file to reduce this component - not sure if that would cause readability issues though